overlay.py

```diff
@@ -45,14 +46,21 @@ def __init__(self, bike, width=1280, height=740, bg:str=None):
 		self.data = DataFactory.create(bike_version)
 		self.device = configs["device"]
 
+		# mqtt client options
 		self.client = mqtt.Client()
 		self.client.on_connect = self._on_connect
 		self.client.on_disconnect = self.on_disconnect
 		self.client.on_log = self.on_log
+		self.client.will_set(
+			f"{str(Camera.online_root)}/{self.device}",
+			dumps({ "online": False }),
+			1,
+			True
+		) # set the camera status to offline if connection breaks
```

Comment:
I would also have the comment above the line that you're doing it just so I can mentally prepare myself with what the next line means.